dnsimple/registrar.go

```diff
@@ -54,10 +54,24 @@ type DomainRegisterRequest struct {
 	EnableAutoRenewal bool `json:"auto_renew,omitempty"`
 }
 
+// DomainRegistration represents the result of a domain renewal call.
+type DomainRegistration struct {
+	ID           int    `json:"id"`
+	DomainID     int    `json:"domain_id"`
+	RegistrantID int    `json:"registrant_id"`
+	Period       int    `json:"period"`
+	State        string `json:"state"`
+	AutoRenew    bool   `json:"auto_renew"`
+	PrivateWhois bool   `json:"private_whois"`
+	PremiumPrice int    `json:"premium_price"`
```

Comment:
Good question. Is it a Float? I've just noticed in https://github.com/dnsimple/dnsimple-elixir/pull/91 that this seems to be a String.

Can you confirm what value it is, and if it is consistent?